docs/manuals/source/TasksAndConcepts/Plugins/StorageDaemonPlugins/ScsicryptoSd.rst.inc

```diff
@@ -81,40 +81,161 @@ The following additional security is needed for the following operating systems:
 Linux (SG_IO ioctl interface):
 
 
-The user running the storage daemon needs the following additional capabilities: :index:`\ <single: Platform; Linux; Privileges>`\
+.. index::
+  single: Platform; Linux; Privileges
+
+To perform the operations required for **scsicrypto**, the programs must either run as user **root** or the additional capability :strong:`CAP_SYS_RAWIO+EP` (see capabilities(7)) must be set.
+The |sd| normally runs as user **bareos**. Running it as **root** is not recommended.
+
+If :command:`bareos-sd` does not have the appropriate capabilities, all other tape operations may still work correctly, but you will get "Unable to perform SG\_IO ioctl" errors.
+
+.. note::
+
+    Since :sinceVersion:`21.0.1: scscicrypto capabilities handling` during package update post-install will react according to file :file:`.enable-cap_sys_rawio` presence in your bareos config dir, setting back the previously set capabilities if present.
```

Comment:
```suggestion
    Since :sinceVersion:`21.0.1: scscicrypto capabilities handling` package installation and upgrade will check for the presence of :file:`.enable-cap_sys_rawio` in your bareos config dir and will configure the required capabilities.
```